### ISSUE TITLE
Is there way to control expiration of succeeded items

### ISSUE THREAD

**Issue by deepakdhirhe** [marcoCasamento/Hangfire.Redis.StackExchange#114]
Hi, 

I observed that succeeded list contains items for last 24hrs efficiently. Is there any way to control expiration of succeeded items to make it expire early or may be to keep it for more than 24hrs?

I saw storage option is having property SucceededListSize. but not sure what impact it will have with redis storage.

Thanks

**Comment by marcoCasamento** [comment]
yes, you need to add a filter that does this and probably you want to apply it to all job in your application, like

`GlobalJobFilters.Filters.Add(new ProlongExpirationTimeAttribute(myTimeSpan));`

search internet for "ProlongExpirationTimeAttribute" or starts from here: https://discuss.hangfire.io/t/how-to-configure-the-retention-time-of-job/34/2
